src/Resources/Users/Referral.ts

```diff
@@ -1,13 +1,12 @@
 import AbstractResource from '../AbstractResource';
-import { HttpClientResponse } from '../../Interfaces/HttpClient';
 
 export default class Referral extends AbstractResource {
     initialise(): void {
         this.resourcePath = '';
         this.defaultAccess = 'member';
     }
 
-    public get: () => HttpClientResponse = this.createMethodFromPartialSpec({
+    public get: Function = this.createMethodFromPartialSpec({
```

Comment:
Ah, no this is now wrong both on param args and return type.

It should be

```suggestion
    public get: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
```

Reply:
Sorry I'm confused as the get method for this endpoint doesn't take any arguments. Am I missing something?

Reply:
It does. It takes an axios config